charts/opentelemetry-demo/values.yaml

```diff
@@ -500,14 +516,118 @@ opentelemetry-collector:
               allowed_origins:
                 - "http://*"
                 - "https://*"
+      otlp/spanmetrics:
+        protocols:
+          grpc:
+            endpoint: "localhost:65535"
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 5s
+              static_configs:
+                - targets: ['0.0.0.0:8888']
+
     exporters:
       otlp:
-        endpoint: '{{ .Release.Name }}-jaeger:4317'
+        endpoint: '{{ include "otel-demo.name" . }}-jaeger:4317'
         tls:
           insecure: true
+      logging:
+      prometheus:
+        endpoint: '0.0.0.0:9464'
+
+    processors:
+      batch:
+      spanmetrics:
+        metrics_exporter: prometheus
+
     service:
+      telemetry:
+        metrics:
+          level: detailed
+          address: 0.0.0.0:8888
       pipelines:
         traces:
-          exporters:
-            - logging
-            - otlp
+          receivers: [otlp]
+          processors: [spanmetrics, batch]
+          exporters: [otlp, logging]
+        metrics:
+          receivers: [otlp, prometheus]
+          processors: [batch]
```

Comment:
Please add the memory_limiter processor here as well.

Reply:
I removed the processor definitions for logs and metrics since they would have been the same as the defaults.